common/changes/@itwin/presentation-common/presentation-add_descriptor_instance_filter_2022-10-27-13-17.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/presentation-common",
+      "comment": "Added instance filter to content descriptor",
```

Comment:
Not sure we can say this when the added attributes are all `@alpha`. Need to either change them to `@beta` or avoid mentioning them in docs.